README.Rmd

```diff
@@ -22,27 +22,25 @@ knitr::opts_chunk$set(
 [![Docs dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://rpolars.github.io/dev)
 <!-- badges: end -->
 
-The goal of this project is to bring the blazingly fast
-[Polars](https://www.pola.rs/) data manipulation library to R. The underlying
-computation engine is written in Rust and this R implementation has no other
-dependencies than R itself (≥ 4.1.0).
+The **polars** package for R gives users access to [a lightning
+fast](https://duckdblabs.github.io/db-benchmark/) Data Frame library written in
+Rust. [Polars](https://www.pola.rs/)' embarrassingly parallel execution, cache
+efficient algorithms and expressive API makes it perfect for efficient data
+wrangling, data pipelines, snappy APIs and so much more. Polars also supports
+"streaming mode" for out-of-memory operations. This allows users to analyze
+datasets many times larger than RAM.
 
 Documentation can be found on the **r-polars**
 [homepage](https://rpolars.github.io).
 
 The primary developer of the upstream Polars project is 
 Ritchie Vink ([@ritchie46](https://github.com/ritchie46)). 
 This R port is maintained by 
-Søren Welling ([@sorhawell](https://github.com/sorhawell)),
-together with other
+Søren Welling ([@sorhawell](https://github.com/sorhawell)) and
```

Comment:
I missed it first time around, but should @eitsupi be mentioned here too?

Reply:
@eitsupi @vincentarelbundock @etiennebacher 

in the current fast pace a lot more names should be added soon :)

Does it make sense for project to openly state a contributor / authorship thumb-rule? and just update whenever a new person meets the criteria?

Reply:
Thanks. Wouldn't it be sufficient for R packages to only mention it in the DESCRIPTION file?

Reply:
I was thinking the people with write/push admin access to the repo itself. (And also think it's a good idea to establish a core group of admins, rather than running through the bottleneck of a single admin.)

But yeah maybe keep it minimal in the README itself

Reply:
I agree. I think this PR is good to merge. I have been late to review and approve PRs. I have started issue #174 to discuss if we can bring in more admins and make the decision structure more flat.

Reply:
@sorhawell sounds great! Click merge whenever ready. Just to add a bit of perspective: My sense is that in most repos, PRs remain open for **weeks** before review or merge. You should not feel pressure to review so quickly. Open source burnout is a real thing, and it's more important to have long term workload sustainability than quick turnaround on PRs. Thanks for all your work!